package.json

```diff
@@ -36,7 +36,7 @@
     "tape-promise": "^3.0.0"
   },
   "engines": {
-    "node": ">=6"
+    "node": ">=4"
```

Comment:
this change doesn't make sense if tests don't run on Node 4

Reply:
I originally tried to get them to run on it, but I see you’ve managed to fix it in https://github.com/pnpm/cmd-shim/commit/7abe91a6a0957084e49d35147db329ee4aee1c90.